packages/reaction-shipping/client/templates/shipping.js

```diff
@@ -109,7 +109,7 @@ Template.editShippingProvider.events({
  * template addShippingProvider events
  */
 Template.addShippingProvider.events({
-  "click [data-event-action=cancelAddShippingProvider]"(event) {
+  "click [data-event-action=cancelAddShippingProvider]" (event) {
```

Comment:
There should be no space here. It was correct before. We are using eslint to inforce / suggest coding style and this is not one of the lines that needs fixing.